examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+    "df_data.head()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "pharmaceutical-chinese",
+   "metadata": {},
+   "source": [
+    "The data contains 16 columns, each of which has different data label. Next, we will use data labeler of the data profiler to predict the label for each column in this tabular dataset. To use only the labeling functionality, other options are disabled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "colored-outside",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set option to run only data labeler\n",
+    "profile_options = dp.ProfilerOptions()\n",
+    "profile_options.set({\"text.is_enabled\": False, \n",
+    "                     \"int.is_enabled\": False, \n",
+    "                     \"float.is_enabled\": False, \n",
+    "                     \"order.is_enabled\": False, \n",
+    "                     \"category.is_enabled\": False, \n",
+    "                     \"datetime.is_enabled\": False,})\n",
+    "\n",
+    "profile = dp.Profiler(data, profiler_options=profile_options)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "agreed-benefit",
+   "metadata": {},
+   "source": [
+    "The results are shown below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "religious-absolute",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get the prediction from data profiler\n",
+    "def get_structured_results(results):\n",
+    "    columns = []\n",
+    "    predictions = []\n",
+    "    for col in results['data_stats']:\n",
+    "        columns.append(col)\n",
+    "        predictions.append(results['data_stats'][col]['data_label'])\n",
+    "\n",
+    "    df_results = pd.DataFrame({'Column': columns, 'Prediction': predictions})\n",
+    "    return df_results\n",
+    "\n",
+    "results = profile.report()    \n",
+    "print(get_structured_results(results))"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "tutorial-movement",
+   "metadata": {},
+   "source": [
+    "The results show that the data profiler is able to detect sensitive information such as datetime, ipv4, or address."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "alleged-association",
+   "metadata": {},
+   "source": [
+    "### Train data labeler from scratch"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "intermediate-lunch",
+   "metadata": {},
+   "source": [
+    "Data labeler can be trained from scratch with the new list of labels. Below, we show an example of training the data labeler on a dataset with labels given as the columns of that dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "found-israel",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the column 'comment' is changed to UNKNOWN, as data labeler requires at least one column with label UNKNOWN\n",
+    "df = data.data.rename({'comment': 'UNKNOWN'}, axis=1)\n",
```

Comment:
There is a flag to do this inplace and I think that would be cleaner than making a new variable

Reply:
which flag? can you point it out?

Reply:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rename.html
you can see inplace = False by default but can be set to true

Reply:
Oh, I thought you meant a flag inside dataprofiler. I thought about this, but this's a small df, and just an example, so i'm not sure we need that. But i'm open to any change if needed.

Reply:
I don't think this can be fixed here, but I do think this is kind of backwards that we have to change a dataset to train upon it. The label mapping should change and the user should be able to set the default label IMO.
@lettergram